content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,125 @@ title: "Security"
 weight: 85
 description: >
   Concepts for keeping your cloud-native workload secure.
+simple_list: true
 ---
+
+This section of the Kubernetes documentation aims to help you learn to run
+workloads more securely, and about the essential aspects of keeping a
+Kubernetes cluster secure.
+
+Kubernetes is based on a cloud-native architecture, and draws on advice from the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practice for
+cloud native information security.
+
+Read [Cloud Native Security and Kubernetes](/docs/concepts/security/cloud-native-security/)
+for the broader context about how to secure your cluster and the applications that
+you're running on it.
+
+## Kubernetes security mechanisms {#security-mechanisms}
+
+Kubernetes includes several APIs and security controls, as well as ways to
+define [policies](#policies) that can form part of how you manage information security.
+
+### Control plane protection
+
+A key security mechanism for any Kubernetes is to
+[control access to the Kubernetes API](/docs/concepts/security/controlling-access).
+
+You can define [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)
```

Comment:
Nit: Recommend calling out TLS first in this paragraph, since it is essential to virtually any effort to secure a kubernetes cluster. Then move to "You can also configure encryption at rest..." because it's a good idea but less critical for many common use cases.

Reply:
(reworded)